RealmSwift/Tests/ObjectCreationTests.swift

```diff
@@ -261,7 +261,7 @@ class ObjectCreationTests: TestCase {
         XCTAssertEqual(stringObjects.count, 1)
         let persistedObject = object.object!
 
-        XCTAssertEqual(persistedObject.intCol, 12)
+        XCTAssertEqual(persistedObject.intCol, 11)
```

Comment:
I don't think it's purposefully testing conflicting values -- that's why I made the comment earlier about it being semi-deterministic (it depends on the order of the column indices)